snuba/datasets/pluggable_entity.py

```diff
@@ -73,25 +73,20 @@ def get_all_join_relationships(self) -> Mapping[str, JoinRelationship]:
     def get_query_pipeline_builder(self) -> QueryPipelineBuilder[ClickhouseQueryPlan]:
         from snuba.pipeline.simple_pipeline import SimplePipelineBuilder
 
-        if is_storage_set_sliced(self.readable_storage.get_storage_set_key()):
-            assert (
-                self.partition_key_column_name is not None
-            ), "partition key column name must be defined for a sliced storage"
-            query_plan_builder: ClickhouseQueryPlanBuilder = (
-                SlicedStorageQueryPlanBuilder(
-                    storage=self.readable_storage,
-                    mappers=self.translation_mappers,
-                    storage_cluster_selector=ColumnBasedStorageSliceSelector(
-                        storage=self.readable_storage.get_storage_key(),
-                        storage_set=self.readable_storage.get_storage_set_key(),
-                        partition_key_column_name=self.partition_key_column_name,
-                    ),
+        storages: List[StorageAndMappers] = [
+            StorageAndMappers(self.readable_storage, self.translation_mappers, False)
+        ]
+        if self.writeable_storage:
+            storages.append(
+                StorageAndMappers(
+                    self.writeable_storage, self.translation_mappers, True
```

Comment:
So are we making an assumption here that the translation mappers are the same no matter the storage?

Reply:
Is that an accurate assumption to make?

Reply:
Correct, this is the assumption at the moment which what we have today. This PR description says that updating the pluggable entity definition to support multiple translation mappers via `List[StorageAndMappers]` will be included in a subsequent PR because those changes will be too large for this one.